models/DataObject/ClassDefinition/Data/Localizedfields.php

```diff
@@ -116,6 +116,16 @@ class Localizedfields extends Data implements CustomResourcePersistingInterface,
      */
     public $fieldDefinitionsCache;
 
+    /**
+     * @var array
+     */
+    public $permissionView = [];
```

Comment:
`permissionView` and `permissionEdit` need to be flushed on sleep.

Reply:
https://github.com/pimcore/pimcore/pull/7487/commits/1b21d73f3c05aec0fe5e7d7b6b55bf6a839f677a